src/test/directives/class-map_test.ts

```diff
@@ -95,6 +105,15 @@ suite('classMap', () => {
     assert.isFalse(el.classList.contains('foo'));
   });
 
+  test('adds classes on SVG elements', () => {
+    const cssInfo = {foo: 0, bar: true, zonk: true};
+    render(svg`<circle class="${classMap(cssInfo)}"></circle>`, container);
+    const el = container.firstElementChild!;
+    assert.isFalse(el.classList.contains('foo'));
```

Comment:
`classList` needs to be replaced with `className.split(' ')...` for SVG elements on IE

Reply:
Done